google-cloud-spanner/acceptance/spanner_helper.rb

```diff
@@ -31,8 +31,16 @@ def SecureRandom.int64
   random_bytes(8).unpack("q")[0]
 end
 
+def emulator_host
+  ENV["SPANNER_EMULATOR_HOST"]
+end
+
+def emulator_enabled?
+  !emulator_host.nil?
+end
+
 # Create shared spanner object so we don't create new for each test
-$spanner = Google::Cloud::Spanner.new
+$spanner = Google::Cloud::Spanner.new(emulator_host: emulator_host)
```

Comment:
If `SPANNER_EMULATOR_HOST` is set, we should not need to pass it as a parameter: 

https://github.com/googleapis/google-cloud-ruby/blob/5f61bf15a744f8d969e6bdae80f57ac8ea150d5b/google-cloud-spanner/test/google/cloud/spanner_test.rb#L325-L341

Reply:
done